pkgs/applications/virtualization/docker/default.nix

```diff
@@ -308,15 +308,15 @@ rec {
   };
 
   docker_26 = callPackage dockerGen rec {
-    version = "26.0.0";
+    version = "26.1.2";
```

Comment:
Looks like 26.1.3 is out? https://docs.docker.com/engine/release-notes/26.1/

Reply:
I can add that tomorrow.

Reply:
Done in https://github.com/NixOS/nixpkgs/commit/0234bac73cf53bee97ccdddef71748be10a88479